deps.macos/80-carla.zsh

```diff
@@ -0,0 +1,64 @@
+autoload -Uz log_debug log_error log_info log_status log_output
+
+## Dependency Information
+local name='carla'
+local version='2.6.0-alpha1'
+local url='https://github.com/falkTX/Carla.git'
+local hash='e5ddd8c064638431d4f731fb7038f7f793630a66'
+
+## Build Steps
+setup() {
+  log_info "Setup (%F{3}${target}%f)"
+  setup_dep ${url} ${hash}
+}
+
+clean() {
+  cd "${dir}"
+
+  if [[ ${clean_build} -gt 0 && -d "build_${arch}" ]] {
+    log_info "Clean build directory (%F{3}${target}%f)"
+
+    rm -rf "build_${arch}"
+  }
+}
+
+config() {
+  autoload -Uz mkcd progress
+
+  log_info "Config (%F{3}${target}%f)"
+
+  args=(
+    ${cmake_flags}
+    -DCARLA_USE_JACK=OFF
```

Comment:
Please add the switch to enable choice between static/dynamic libraries.

If code is required by a single module we default to using static libraries and only chose dynamic libraries as an exception (e.g., x264, srt, and librist, which are used by multiple consumers).

Reply:
the carla-related OBS module has 2 obs plugins inside - carla-bridge and carla-patchbay. so it makes sense to have carla-utils as shared library.